device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/BALANCED/buffers_defaults_t0.j2

```diff
@@ -8,7 +8,7 @@
             "size": "165660324",
```

Comment:
I believe these values should now be reverted back to reflect the production bcm config, e.g. `121964196` here

Reply:
not for the pool, these values are the latest recommended ones.

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/19570/commits/b62e111370a9c11c0efaa4d55ea5106af858049f#diff-38141bc38cf22f25e51d1afa26023349fcb218e27d0c46e8ada19c6c9081111cL2251-L2252

I'm not sure how `165660324` was derived in the second commit, but I believe this change to the SHARED_LIMIT_CELLS needs to be reflected? (240087 * 254 * 2 = 121964196)

Reply:
I think we are aligned from calculation perspective, however if you check the sai api values that shared by broadcom. the value is indeed this....